src/movies.js

```diff
@@ -1,15 +1,132 @@
 /* eslint no-restricted-globals: 'off' */
 
-// Iteration 1: Ordering by year - Order by year, ascending (in growing order)
+// Iteration 1: Ordering by year - Order by year, ascending (in growing order) 
+
+
+function orderByYear (movies) {
+
+
+    const newMovies = [...movies];
+    
+    newMovies.sort(function(movieA, movieB){
+        if(movieA.year < movieB.year){
+            return -1;
+        }
+        if(movieA.year > movieB.year){
+            return 1;
+        }
+        if(movieA.year === movieB.year){
+            if(movieA.title < movieB.title){
+                return -1
+            }
+            if(movieA.title > movieB.title){
+                return 1
+            }
+            if(movieA.title === movieB.title){
+                return 0
+            }
+            return 0;
```

Comment:
This return is redundant as you are already considering all possible cases and returning 0 when titles are the same. So you could remove it.